cartographer/io/serialization_format_migration.cc

```diff
@@ -18,284 +18,200 @@
 
 #include <vector>
 
-#include "absl/container/flat_hash_map.h"
+#include "cartographer/common/config.h"
+#include "cartographer/common/configuration_file_resolver.h"
+#include "cartographer/common/lua_parameter_dictionary.h"
+#include "cartographer/common/thread_pool.h"
+#include "cartographer/io/internal/mapping_state_serialization.h"
+#include "cartographer/io/proto_stream_deserializer.h"
 #include "cartographer/mapping/3d/submap_3d.h"
+#include "cartographer/mapping/id.h"
+#include "cartographer/mapping/internal/3d/pose_graph_3d.h"
 #include "cartographer/mapping/internal/3d/scan_matching/rotational_scan_matcher.h"
+#include "cartographer/mapping/map_builder.h"
+#include "cartographer/mapping/map_builder_interface.h"
+#include "cartographer/mapping/pose_graph.h"
 #include "cartographer/mapping/probability_values.h"
-#include "cartographer/mapping/proto/internal/legacy_serialized_data.pb.h"
-#include "cartographer/mapping/proto/internal/legacy_submap.pb.h"
```

Comment:
You can remove the protos in the `mapping/proto/internal` folder. I added them in #1314 for the sole purpose of this migration tool.

Reply:
Thanks for the suggestion. I'll do this in a follow up PR.